FRUI/FRUI/UIElements/DropDownButton/FRActionSheetButton.swift

```diff
@@ -0,0 +1,145 @@
+// 
+//  FRDropDownActionButton.swift
+//  FRUI
+//
+//  Copyright (c) 2019 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import UIKit
+
+@objc
+public protocol FRActionSheetProtocol {
+    @objc
+    func selectedItem(index: Int, item: String)
+}
+
```

Comment:
I don't know if it's intentional, but there is no inline documentation for most of FRUI classes. If it is intentional, please ignore this comment.

Reply:
Some UI elements in FRUI SDK are missing documentations which is intended. Will add more documentation and proper guideline for UI framework soon.